js/everything.js

```diff
@@ -580,32 +573,32 @@ $(function() {
     // How do we know if the value is truly invalid?
     // Preload the form from the URL
     var currentVideoID = getCurrentVideoID();
-    if (currentVideoID) {
+    if (currentVideoID)
+    {
```

Comment:
Please put the brace at the end of the previous line (with a space after the `)`)